crates/sui-graphql-rpc/src/types/transaction_block/filter.rs

```diff
@@ -94,30 +103,37 @@ impl TransactionBlockFilter {
     }
 
     /// If we don't query a lookup table that has a denormalized sender column, we need to
-    /// explicitly sp
+    /// explicitly specify the sender.
```

Comment:
even though this was my comment, it took me some time to wrap my head around this. I recall now that `explicit_sender` is called when constructing the subqueries to determine whether to explicitly add a subquery against `tx_senders`